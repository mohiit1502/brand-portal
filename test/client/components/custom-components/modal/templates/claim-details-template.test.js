// /* eslint-disable filenames/match-regex, no-unused-vars, no-undef */
// import React from "react";
// import {Provider} from "react-redux";
// import {BrowserRouter} from "react-router-dom";
// import renderer from "react-test-renderer";
// import Cookies from "electrode-cookies";
// import {testStore} from "../../../../../../src/client/utility/TestingUtils";
// import ClaimDetailsTemplate from "../../../../../../src/client/components/custom-components/modal/templates/claim-details-template";
// import profile from "../../../../mocks/userProfile";
// import Http from "../../../../../../src/client/utility/Http";
// import MODALSMETA from "../../../../../../src/client/config/contentDescriptors/modals-meta";
// import FORMFIELDMETA from "../../../../../../src/client/config/formsConfig/form-field-meta";
// import {mount} from "enzyme";
//
// let store;
// jest.mock("electrode-cookies");
//
// const setUpRender = () => {
//   const mockStore = {
//     content: {
//       metadata: {...MODALSMETA, ...FORMFIELDMETA},
//     },
//     user: {profile}
//   };
//   const data = {
//     firstName: "test-fname",
//     lastName: "test-lname",
//     type: "test-counterfeit",
//     items: [{}]
//   }
//   store = testStore(mockStore);
//   return renderer.create(<Provider store={store}><ClaimDetailsTemplate data={data}/></Provider>);
// };
//
// describe("ClaimDetailsTemplate test container", () => {
//   let wrapper;
//   beforeEach(() => {
//     jest.restoreAllMocks();
//     jest.resetAllMocks();
//     Cookies.get.mockImplementation(jest.fn());
//   });
//
//   it("should render the ClaimDetailsTemplate successfully", () => {
//     wrapper = setUpRender();
//     wrapper.root.findByProps({ className: "close text-white" }).props.onClick();
//     wrapper.root.findByProps({ className: "btn btn-sm btn-primary submit-btn px-3 mx-3" }).props.onClick();
//     wrapper = renderer.create(<Provider store={store}><ClaimDetailsTemplate /></Provider>);
//     expect(wrapper.toJSON()).toMatchSnapshot();
//   });
// });
