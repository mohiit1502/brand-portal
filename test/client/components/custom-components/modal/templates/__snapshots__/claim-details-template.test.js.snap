// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClaimDetailsTemplate test container should render the ClaimDetailsTemplate successfully 1`] = `
<div
  className="modal claim-details-modal show"
  id="singletonModal"
  role="dialog"
  tabIndex="-1"
>
  <div
    className="modal-dialog modal-dialog-centered modal-xl"
    role="document"
  >
    <div
      className="modal-content"
    >
      <div
        className="modal-header align-items-center px-4"
      >
        Claim Details
        <button
          aria-label="Close"
          className="close text-white"
          onClick={[Function]}
          type="button"
        >
          <span
            aria-hidden="true"
          >
            Ã—
          </span>
        </button>
      </div>
      <div
        className="modal-body px-0 pb-0 text-left  loader"
        style={
          Object {
            "minHeight": "25rem",
          }
        }
      >
        <p
          style={
            Object {
              "padding": "1rem",
            }
          }
        >
          Getting Claim Details....
        </p>
      </div>
    </div>
  </div>
</div>
`;
